Mathlib/Combinatorics/Additive/FreimanHom.lean

```diff
@@ -0,0 +1,347 @@
+    rw [Pi.div_def, prod_map_div, prod_map_div, h₁.map_prod_eq_map_prod hsA htA hs ht h,
+      h₂.map_prod_eq_map_prod hsA htA hs ht h]
+
+end DivisionCommMonoid
+
+#noalign add_freiman_hom
+#noalign freiman_hom
+#noalign add_freiman_hom_class
+#noalign freiman_hom_class
+#noalign freiman_hom.fun_like
+#noalign add_freiman_hom.fun_like
+#noalign freiman_hom.freiman_hom_class
+#noalign add_freiman_hom.freiman_hom_class
+#noalign freiman_hom.to_fun_eq_coe
+#noalign add_freiman_hom.to_fun_eq_coe
+#noalign freiman_hom.ext
+#noalign add_freiman_hom.ext
+#noalign freiman_hom.coe_mk
+#noalign add_freiman_hom.coe_nat_eq_mk
+#noalign freiman_hom.mk_coe
+#noalign add_freiman_hom.mk_coe
+#noalign add_freiman_hom.id
+#noalign freiman_hom.id_apply
+#noalign freiman_hom.comp
+#noalign add_freiman_hom.comp
+#noalign freiman_hom.coe_comp
+#noalign add_freiman_hom.coe_comp
+#noalign freiman_hom.comp_apply
+#noalign add_freiman_hom.comp_apply
+#noalign freiman_hom.comp_assoc
+#noalign add_freiman_hom.comp_assoc
+#noalign freiman_hom.cancel_right
+#noalign add_freiman_hom.cancel_right
+#noalign freiman_hom.cancel_right_on
+#noalign add_freiman_hom.cancel_right_on
+#noalign freiman_hom.cancel_left_on
+#noalign add_freiman_hom.cancel_left_on
+#noalign freiman_hom.comp_id
+#noalign add_freiman_hom.comp_id
+#noalign freiman_hom.id_comp
+#noalign add_freiman_hom.id_comp
+#noalign freiman_hom.const
+#noalign add_freiman_hom.const
+#noalign freiman_hom.const_apply
+#noalign add_freiman_hom.const_apply
+#noalign freiman_hom.const_comp
+#noalign add_freiman_hom.const_comp
+#noalign freiman_hom.one_apply
+#noalign add_freiman_hom.zero_apply
+#noalign freiman_hom.one_comp
+#noalign add_freiman_hom.zero_comp
+#noalign freiman_hom.mul_apply
+#noalign add_freiman_hom.add_apply
+#noalign freiman_hom.mul_comp
+#noalign add_freiman_hom.add_comp
+#noalign freiman_hom.inv_apply
+#noalign add_freiman_hom.neg_apply
+#noalign freiman_hom.inv_comp
+#noalign add_freiman_hom.neg_comp
+#noalign freiman_hom.div_apply
+#noalign add_freiman_hom.sub_apply
+#noalign freiman_hom.div_comp
+#noalign add_freiman_hom.sub_comp
+#noalign freiman_hom.comm_monoid
+#noalign add_freiman_hom.add_comm_monoid
+#noalign freiman_hom.comm_group
+#noalign add_freiman_hom.add_comm_group
+#noalign monoid_hom.freiman_hom_class
+#noalign add_monoid_hom.freiman_hom_class
+#noalign monoid_hom.to_freiman_hom
+#noalign add_monoid_hom.to_add_freiman_hom
+#noalign monoid_hom.to_freiman_hom_coe
+#noalign add_monoid_hom.to_freiman_hom_coe
+#noalign monoid_hom.to_freiman_hom_injective
+#noalign add_monoid_hom.to_freiman_hom_injective
+#noalign freiman_hom.to_freiman_hom
+#noalign add_freiman_hom.to_add_freiman_hom
+#noalign freiman_hom.freiman_hom_class_of_le
+#noalign add_freiman_hom.add_freiman_hom_class_of_le
+#noalign freiman_hom.to_freiman_hom_coe
+#noalign add_freiman_hom.to_add_freiman_hom_coe
+#noalign freiman_hom.to_freiman_hom_injective
+#noalign add_freiman_hom.to_freiman_hom_injective
```

Comment:
This just feels like noise at this point: Nobody ever used these in mathlib nor will have an unported project depending on them.

Should I just remove?

Reply:
I don't care too much. Let's leave them in for now...